specifications/ticket-api.yaml

```diff
@@ -0,0 +1,401 @@
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/query-parameters'
+          application/xml:
+            schema:
+              $ref: '#/components/schemas/query-parameters'
+  /tickets/confirm:
+    post:
+      tags:
+        - purchase
+      summary: Confirm transaction
+      description: Confirm provided transaction and receive a ticket product
+      operationId: condirmTransaction
+      responses:
+        '200':
+          description: Successful operation
+          content:
+            application/xml:
+              schema:
+                $ref: '#/components/schemas/ticket-response'
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ticket-response'
+        '400':
+          description: Invalid parameters
+        '403':
+          description: Transaction denied
+        '404':
+          description: Service not found
+      requestBody:
+        content:
+          application/xml:
+            schema:
+              $ref: '#/components/schemas/confirm-parameters'
+          application/json:
+            schema:
+              $ref: '#/components/schemas/confirm-parameters'
+
+components:
+  schemas:
+    geo-point:
+      type: object
+      properties:
+        lat:
+          type: number
+          description: Latitude of the point (WGS84)
+        lon:
+          type: number
+          description: Longitude of the point (WGS84)
+    price:
+      type: object
+      properties:
+        currency:
+          type: string
+          enum:
+            - EUR
+        amount:
+          type: string
+    identity:
+      type: object
+      properties:
+        type:
+          type: string
+          enum:
+            - PKI
+            - X.509
+            - OAUTH2
+            - CREDIT
+            - PHONE
+            - NAME
+        issuer:
+          type: string
+          format: uri
+        data:
+          type: string
+    query-parameters:
+      type: object
+      properties:
+        ids:
+          description: List of identities the client can authenticate with
+          type: array
+          xml:
+            name: id
+          items:
+            $ref: '#/components/schemas/identity'
+          example:
+            - type: X.509
+              issuer: trusted.org
+              data: 'https://maas.example.com/clients/12345.crt'
+            - type: PHONE
+              issuer: ''
+              data: +000 12 345 6789
+            - type: NAME
+              issuer: ''
+              data: Name Example
+            - type: OAUTH2
+              issuer: maas.example.com
+              data: client12345@maas.example.com
+            - type: OAUTH2
+              issuer: students.example.com
+              data: student12345@students.example.com
+            - type: CREDIT
+              issuer: credit.example.com
+              data: credit-visa
+        supportedTickets:
```

Comment:
Would be good to use a unified casing style and change this to `supported-tickets`